wiki/markdown/docker.md

```diff
@@ -0,0 +1,81 @@
+# Docker
+
+In RemRem-Publish source code a Dockerfile is provided which helps the developer or user to build the local RemRem-Publish source code repository changes to a Docker image.
+With the Docker image user can try-out the RemRem-Publish on a Docker Host or in a Kubernetes cluster.
+
+## Requirements
+- Docker 
+
+
+  Linux: https://docs.docker.com/install/linux/docker-ce/ubuntu/
+
+  
+  Windows: https://docs.docker.com/docker-for-windows/install/
+
+
+## Follow these step to build the Docker image.
+
+1. Execute maven install to build the all components and RemRem-Publish
+`mvn install -DskipTests`
+
+This will produce a war file in the "publish-service/target" folder.
+
+2. Change to publish-service directory: 
+`cd publish-service`
+
+3. Build the Docker image with the war file that was produced from previous step: 
+
+
+`docker build -t remrem-publish:0.6.9 --build-arg URL=./target/publish-service-0.6.9.war -f src/main/docker/Dockerfile .` 
+
+
+Now docker image has build with tag "remrem-publish:0.6.9"
+
+## Run Docker image on local Docker Host
+To run the produced docker image on the local Docker host, execute this command: 
+
+
+`docker run -p 8080:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG remrem-publish:0.6.9`
+
+RabbitMq and other RemRem-Publish required components need to running and configured via these application properties that is provided to the docker command above. See the application.properties file for all available/required properties:
```

Comment:
what does "**these** application properties" refer to? Remove the "these" and I think it becomes more clear to read.

Reply:
Fixed.